rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -85,7 +85,10 @@ void main_window::Init()
 
 	// for highdpi resize toolbar icons and height dynamically
 	// choose factors to mimic Gui-Design in main_window.ui
-	const int toolBarHeight = menuBar()->sizeHint().height() * 1.5;
+	int toolBarHeight = menuBar()->sizeHint().height() * 1.5;
+	if (toolBarHeight <= 0) {
```

Comment:
could you move { to new line please? :)
else my whole style adjustment PR would be flawed xD

Reply:
Sure, it's done.

Reply:
btw, shouldn't there be an explicit static_cast to int?